app/views/apo/_form.html.erb

```diff
@@ -103,7 +103,7 @@
 
   <div class="form-group">
     <%= f.label :default_rights, "Default Object Rights" %>
-    <%= f.select :default_rights, options_for_select(Constants::REGISTRATION_RIGHTS_OPTIONS, @form.default_rights), {}, class: 'form-control' %>
+    <%= f.select :default_rights, options_for_select(Constants::REGISTRATION_RIGHTS_OPTIONS, @form.default_object_rights), {}, class: 'form-control' %>
```

Comment:
It's conventional for the existing value to come from a method with the same name as the form field (as they were previously).  Can you align these?

Reply:
I think you might want to keep the ERB as it was and change this line: https://github.com/sul-dlss/argo/blob/48b8c859a723eba4092f50870c6de1d6e2560ffc/app/forms/apo_form.rb#L38 to call the `default_object_rights` method you defined.

Reply:
Done, also made `default_object_rights` private and adjusted the tests accordingly